pkg/providers/ovirt/client/ovirt-client.go

```diff
@@ -151,6 +153,44 @@ func (client *richOvirtClient) StopVM(id string) error {
 	}
 }
 
+// StartVM requests VM start and doesn't wait for it to be UP
+func (client *richOvirtClient) StartVM(id string) error {
+	vmService := client.connection.SystemService().VmsService().VmService(id)
+
+	vmResponse, _ := vmService.Get().Send()
+	vm, vmAvailable := vmResponse.Vm()
+	if !vmAvailable {
+		return fmt.Errorf("Failed to start vm %s, vm is not available", id)
+	}
+
+	// Request the VM startup if VM is not UP
+	if status, _ := vm.Status(); status != ovirtsdk.VMSTATUS_UP {
+		_, err := vmService.Start().Send()
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+// RenameVM updates VM's name
+func (client *richOvirtClient) RenameVM(id string, name string) error {
+	vmService := client.connection.SystemService().VmsService().VmService(id)
+
+	vmResponse, _ := vmService.Get().Send()
+	vm, vmAvailable := vmResponse.Vm()
+	if !vmAvailable {
+		return fmt.Errorf("Failed to rename vm %s, vm is not available", id)
+	}
+
+	vm.SetName(name)
```

Comment:
This is not recomended way of updating vm in oVirt API. The comended way is to just send the parameters you want to update not every parameter. So would be better to change it to something like:

```
newVm, err := ovirtsdk.NewVmBuilder().Name(name).Build()
_, err := vmService.Update().Vm(newVm).Send()
```

Reply:
/release-note-none

Reply:
Thanks!